src/ralph/data_importer/management/commands/demodata.py

```diff
@@ -0,0 +1,104 @@
+from ralph.back_office.models import BackOfficeAssetStatus
+from ralph.back_office.tests.factories import BackOfficeAssetFactory
+from ralph.data_center.models.choices import DataCenterAssetStatus
+from ralph.data_center.tests.factories import (
+    AccessoryFactory,
+    DataCenterAssetFactory,
+    RackAccessoryFactory,
+    RackFactory,
+    ServerRoomFactory
+)
+from ralph.licences.tests.factories import LicenceFactory
+from ralph.supports.tests.factories import SupportFactory
+
+
+class Command(BaseCommand):
+
+    help = "Generating demo data."
+    all_apps = ['backoffice', 'datacenter', 'licences', 'supports']
+    object_limit = 30
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-a', '--apps',
+            choices=self.all_apps + ['all'],
+            default='all',
+            dest='apps',
+            help='One or more apps separated by comma.'
+        )
+
+    def generate_data_center(self):
+        self.stdout.write('Generating Data center assets')
+        data_center_status = DataCenterAssetStatus()
+        status_count = len(data_center_status)
+        per_page = self.object_limit / status_count
+
+        for i in range(4):
+            server_room = ServerRoomFactory()
+            for j in range(self.object_limit):
+                rack = RackFactory(server_room=server_room)
+                accessory = AccessoryFactory()
+                RackAccessoryFactory(rack=rack, accessory=accessory)
+
+        for status_id, name in data_center_status:
+            for i in range(int(per_page)):
+                DataCenterAssetFactory(status=status_id)
+
+    def generate_back_office(self):
+        self.stdout.write('Generating Back Office assets')
+        back_office_status = BackOfficeAssetStatus()
+        status_count = len(back_office_status)
+        per_page = self.object_limit / status_count
+
+        for status_id, name in back_office_status:
+            for i in range(int(per_page)):
+                BackOfficeAssetFactory(
+                    status=status_id,
+                    user=self.get_user(),
+                    owner=self.get_user(),
+                )
+
+    def generate_licence(self):
+        self.stdout.write('Generating Licences')
+        for i in range(self.object_limit):
+            LicenceFactory()
+
+    def generate_users(self):
+        user_ids = []
+        for i in range(self.object_limit):
+            user_ids.append(UserFactory().id)
+        return user_ids
+
+    def get_user(self):
+        return self.users[choice(self.users_id)]
+
+    def generate_support(self):
+        self.stdout.write('Generating Supports')
+        for i in range(self.object_limit):
+            SupportFactory()
+
+    def handle(self, *args, **options):
+        apps = options.get('apps').split(',')
+        if 'all' in apps:
+            apps = self.all_apps
+
+        self.users_id = self.generate_users()
+        self.users = RalphUser.objects.in_bulk(self.users_id)
```

Comment:
Users aren't already created by this line:
`user_ids.append(UserFactory().id)` from `generate_users(..)`
?